src/main/java/frc/robot/commands/AutonModes.java

```diff
@@ -0,0 +1,198 @@
+                () -> driveBaseSubsystem.resetOdometry(oneBallTrajectories[0].getInitialPose())));
+
+    twoBallParallel =
+        new ParallelDeadlineGroup(
+            twoBallRamseteCommands[0].beforeStarting(
+                () -> driveBaseSubsystem.resetOdometry(twoBallTrajectories[0].getInitialPose())),
+            new IntakeForwardCommand(intakeSubsystem));
+    twoBallCommand =
+        new SequentialCommandGroup(
+            new WaitCommand(Constants.delaytaxi),
+            twoBallParallel,
+            twoBallRamseteCommands[1], // goes to fender
+            new WaitCommand(Constants.delayshot),
+            // new ShooterPrime(shooterSubsystem, limelightSubsystem, cdsSubsystem)
+            new ShooterPressed(shooterSubsystem, limelightSubsystem, cdsSubsystem, true));
+  }
+
+  private Trajectory[] getTrajectories(String routeString) {
+    // method to intialize multiple trajectories in one auton route
+    String routeName = routeString; // TODO: replace route for the correct one on pathweaver
+    File f =
+        new File(
+            "Autos/" + routeName); // get file with all the names of the path for the auton routine
+    Scanner fileScanner = null;
+
+    try {
+      fileScanner = new Scanner(f);
+    } catch (FileNotFoundException e) {
+      System.out.println("Scanner for file: " + "Autos/" + routeName + " unable to be created");
+    }
+
+    List<Trajectory> trajectoryList =
+        new ArrayList<
+            Trajectory>(); // use list to take in all trajectories, convert it into array later
+
+    while (fileScanner.hasNext()) {
+      String pathName = "paths/" + fileScanner.nextLine().split("\\.")[0] + ".wpilib.json";
+      Path path =
+          Filesystem.getDeployDirectory()
+              .toPath()
+              .resolve(pathName); // goes to scr/main/deploy/paths
+      Trajectory trajectory;
+      try {
+        trajectory = TrajectoryUtil.fromPathweaverJson(path);
+        trajectoryList.add(trajectory);
+      } catch (IOException e) {
+        System.out.println("Trajectory: " + pathName + " not created.");
+      }
+    }
+    fileScanner.close();
+
+    Trajectory[] trajectoryArray;
+    trajectoryArray = new Trajectory[trajectoryList.size()];
+    trajectoryArray = trajectoryList.toArray(trajectoryArray);
+    return trajectoryArray;
+  }
+
+  private RamseteCommand[] getRamseteCommands(Trajectory[] trajectories) {
+    RamseteCommand[] ramseteCommands = new RamseteCommand[trajectories.length];
+
+    for (int i = 0; i < ramseteCommands.length; i++) {
+      ramseteCommands[i] =
+          new RamseteCommand(
+              trajectories[i],
+              driveBaseSubsystem::getPose,
+              new RamseteController(
+                  Constants.ramseteB,
+                  Constants.ramseteZeta), // ramsete follower to follow trajectory
+              Constants.driveKinematics,
+              driveBaseSubsystem::acceptWheelSpeeds,
+              driveBaseSubsystem);
```

Comment:
```suggestion
              driveBaseSubsystem).beforeStarting(
                  () -> driveBaseSubsystem.resetOdometry(trajectories[i].getInitialPose()));
```

Then its bundled into all the ramsete commands and you don't have to redo it for each in initializeCommandGroups